doc/telemetry-dev-env/Readme.md

```diff
@@ -0,0 +1,46 @@
+Build the image:
+```
+docker build -t telemetry-dev .
+```
+
+Remove container if it exist already:
+```
+docker rm telemetry-dev-env
+```
+Create personal fork repo from sonic-telemetry from git
+
+Creatd directory 'src' and Clone git repos in it:
+```
+git clone https://<your_github_username>@github.com/<your_github_username>/sonic-telemetry.git 
+git clone https://<your_github_username>@github.com/<your_github_username>/sonic-mgmt-common.git
+```
+
+Create a new container using the image:
+```
+docker run -it -v ~/src:/src --name telemetry-dev-env telemetry-dev
```

Comment:
you may want to add "--net=host" to make the other machine to access the telemetry server.